### PR TITLE
feat(DIA-203): adjust Artist header image ratio and minor font and spacing values

### PR DESCRIPTION
**Pull request by araujobarret**
This PR resolves [DIA-203] 

### Description

Changes on the Artist page:
- Adjust the Artist's image size to a shorter one
- Adjust the font size of the Artist description
- Bring back the Artworks counter to the Artist's Artwork grid

Artist without verified representation
Screenshot 2023-10-26 at 09 07 32

Artist with verified representation
Screenshot 2023-10-26 at 09 07 08

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- feat: artist screen changes including image ratio, artwork counter, font spacing, and spaces adjusts - araujobarret

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[DIA-203]: https://artsyproduct.atlassian.net/browse/DIA-203?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Components/Artist/ArtistHeader.tsx [hunk 1]
```diff
@@ -36,8 +38,10 @@ export const useArtistHeaderImageDimensions = () => {
   const { width } = useScreenDimensions()
   const isTablet = isPad()
 
-  const height = isTablet ? ARTIST_IMAGE_TABLET_HEIGHT : width
-  const aspectRatio = width / height
+  const height = isTablet
```
Comment (gkartalis) [inline src/app/Components/Artist/ArtistHeader.tsx:41]: minor and unrelated - but can we change the `isPad()` to be `isTablet` from `import { isTablet } from "react-native-device-info"`?

I can also open a follow up PR to clean the rest of the references of the outdated `isPad` func

File: src/app/Scenes/Artist/Artist.tsx [hunk 2]
```diff
@@ -138,7 +137,8 @@ export const Artist: React.FC<ArtistProps> = (props) => {
           }}
           BelowTitleHeaderComponent={renderBelowTheHeaderComponent}
         >
-          <Tabs.Tab name="Artworks" label="Artworks">
+          {/* @ts-ignore */}
+          <Tabs.Tab name="Artworks" label="Artworks" style={{ backgroundColor: "pink" }}>
```
Comment (gkartalis) [inline src/app/Scenes/Artist/Artist.tsx:141]: 😳 was this intentional?

**Review by gkartalis — DISMISSED** [review]

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (feat: artist screen changes including image ratio, artwork counter, font spacing, and spaces adjusts - araujobarret)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 14785a79cd33f4f9f31a80b69be7a0869b76273e
</p>

File: src/app/Scenes/Artist/Artist.tsx (hunk 2 above)
Comment (araujobarret) [reply src/app/Scenes/Artist/Artist.tsx:141]: ops, no was a test to see if we were passing styles! good catch!

File: src/app/Components/Artist/ArtistHeader.tsx (hunk 1 above)
Comment (araujobarret) [reply src/app/Components/Artist/ArtistHeader.tsx:41]: I can do that right now, I thought the `isPad` was our way to go 👍

Comment (araujobarret) [reply src/app/Components/Artist/ArtistHeader.tsx:41]: changed in https://github.com/artsy/eigen/pull/9490/commits/cc765b6f62763ff58afaa8d53bd1c813f6ef92f1 👍

**Review by gkartalis — APPROVED** [review]

**Pull request merged**